### ISSUE TITLE
[Accessibility] Reports: Insufficient color contrast in custom date text

### ISSUE THREAD

**Issue by iamdharmesh** [woothemes/woocommerce#61682]
### Prerequisites

- [x] I have carried out troubleshooting steps and I believe I have found a bug.
- [x] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

Original Report: https://linear.app/a8c/issue/WOOVEND-120/accessibility-insufficient-color-contrast-vendor-reports-orders-24 (Reported for the Product Vendors)

The Dates custom text fails color contrast against the background color. See color contrast analysis: https://webaim.org/resources/contrastchecker/?fcolor=777777&bcolor=F5F5F5 

<img width="1593" height="718" alt="Image" src="https://github.com/user-attachments/assets/32de7f71-d500-45c5-8b01-a8dcf8ea278d" />

**Component or Page:**
Reports: Orders

**Relevant code snippets:**
```css
.woocommerce-reports-wide .postbox div.stats_range ul li.custom div input.range_datepicker, .woocommerce-reports-wide .postbox div.stats_range ul li.custom form input.range_datepicker, .woocommerce-reports-wide .postbox h3.stats_range ul li.custom div input.range_datepicker, .woocommerce-reports-wide .postbox h3.stats_range ul li.custom form input.range_datepicker, .woocommerce-reports-wrap .postbox div.stats_range ul li.custom div input.range_datepicker, .woocommerce-reports-wrap .postbox div.stats_range ul li.custom form input.range_datepicker, .woocommerce-reports-wrap .postbox h3.stats_range ul li.custom div input.range_datepicker, .woocommerce-reports-wrap .postbox h3.stats_range ul li.custom form input.range_datepicker {
    padding: 0;
    margin: 0 10px 0 0;
    background: 0 0;
    border: 0;
    color: #777;
    text-align: center;
    box-shadow: none;
}
```

**Recommended fix:**
Change the colors to have a minimum color contrast of at least 4.5:1.

### Expected behavior

The custom date text color should have a minimum color contrast of at least 4.5:1.

### Actual behavior

The custom date text color have a color contrast of 4.1:1.

### Steps to reproduce

1. Go to `WooCommerce` > Reports
2. Select custom dates and verify that it don't have sufficient color contrast.

### WordPress Environment

<details>

<summary>WordPress Environment</summary>

<details><summary>System Status Report</summary>

```
### WordPress Environment ###

WordPress address (URL): [Redacted]
Site address (URL): [Redacted]
WC Version: 10.3.3
Legacy REST API Package Version: The Legacy REST API plugin is not installed on this site.
Action Scheduler Version: ✔ 3.9.3
Log Directory Writable: ✔
WP Version: 6.8.3
WP Multisite: –
WP Memory Limit: 256 MB
WP Debug Mode: ✔
WP Cron: ✔
Language: en_US
External object cache: –

### Server Environment ###

Server Info: nginx/1.26.1
Server Architecture: Darwin 24.6.0 arm64
PHP Version: 8.4.10
PHP Post Max Size: 1,000 MB
PHP Time Limit: 1200
PHP Max Input Vars: 4000
cURL Version: 8.7.1
(SecureTransport) LibreSSL/3.3.6

SUHOSIN Installed: –
MySQL Version: 8.0.35
Max Upload Size: 300 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

[REDACTED]

### Post Type Counts ###

attachment: 94
breakdance_template: 7
custom_css: 1
event_ticket: 160
event_ticket_email: 19
mailpoet_page: 1
nav_menu_item: 17
page: 39
post: 20
postman_sent_mail: 64
product: 152
product_variation: 79
revision: 146
shop_coupon: 5
shop_order: 2173
shop_order_placehold: 12
shop_order_refund: 55
shop_subscription: 129
user_request: 2
wc_booking: 6
wc_membership_plan: 1
wc_order_status: 12
wc_user_membership: 1
wgc_subscription: 20
wp_font_face: 33
wp_font_family: 12
wp_global_styles: 6
wp_navigation: 2
wp_template_part: 3
wpforms: 1

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ✔

### Active Plugins (7) ###

Email Log: by WebFactory Ltd – 2.61
Loco Translate: by Tim Whitlock – 2.8.0
WooCommerce EU VAT Number: by WooCommerce – 3.0.2
Braintree for WooCommerce Payment Gateway: by WooCommerce – 3.4.2
WooCommerce Pre-Orders: by WooCommerce – 2.3.0 (update to version 2.3.1 is available)
WooCommerce Product Vendors: by WooCommerce – 2.4.7
WooCommerce: by Automattic – 10.3.3

### Inactive Plugins (31) ###

AutomateWoo: by WooCommerce – 6.0.8 (update to version 6.1.18 is available)
GoCardless for WooCommerce: by GoCardless – 2.9.4 (update to version 2.9.8 is available)
Query Monitor: by John Blackbourn – 3.20.0
Restricted Site Access: by Jake Goldman
10up
Oomph – 7.3.4

User Switching: by John Blackbourn – 1.10.0
WooCommerce.com Update Manager: by Automattic – 1.0.3
WooCommerce Box Office: by WooCommerce – 1.3.3 (update to version 1.3.7 is available)
WooCommerce Bulk Stock Management: by WooCommerce – 2.3.1 (update to version 2.3.2 is available)
WooCommerce Currency Converter: by WooCommerce – 1.6.29 (update to version 2.2.5 is available)
WooCommerce Deposits: by WooCommerce – 2.3.9 (update to version 2.4.1 is available)
WooCommerce Elavon Converge EU Gateway: by Elavon – 1.16.0
WooCommerce EU VAT Number SOAP Fix.: by  –
WooCommerce Eway Payment Gateway: by WooCommerce – 3.6.0 (update to version 3.9.2 is available)
WooCommerce Framework Plugin TODO: plugin name: by SkyVerge – 1.0.0 TODO: plugin version
WooCommerce GoCardless Gateway: by GoCardless – 2.9.0
WooCommerce Order Barcodes: by WooCommerce – 1.7.3 (update to version 1.9.7 is available)
WooCommerce Order Status Manager: by SkyVerge – 1.13.4 (update to version 1.15.6 is available)
WooCommerce Payfast Gateway: by WooCommerce – 1.7.3
WooCommerce PayPal Payments: by PayPal – 3.0.6 (update to version 3.2.1 is available)
WooCommerce Product CSV Import Suite: by WooCommerce – 1.10.77 (update to version 1.10.80 is available)
WooCommerce Purchase Order Payment Gateway: by WooCommerce – 1.5.3 (update to version 1.5.7 is available)
WooCommerce SnapScan Gateway: by WooCommerce – 1.2.7
WooCommerce Square: by WooCommerce – 4.7.2 (update to version 5.1.0 is available)
WooCommerce Stripe Gateway: by Stripe – 9.9.2
WooCommerce Subscriptions: by WooCommerce – 7.7.0 (update to version 8.0.0 is available)
WooCommerce Subscriptions CSV Importer and Exporter: by Prospress Inc – 2.2.0
WooCommerce WooCommerce GoCardless Gateway Test plugin: by  –
WooCommerce Xero Integration: by WooCommerce – 1.9.7 (update to version 1.9.8 is available)
WooPayments: by WooCommerce – 10.1.1
WP Crontrol: by John Blackbourn – 1.19.3
WP Rollback: by WP Rollback – 3.0.8

### Must Use Plugins (1) ###

ngrok.php: by  –

### Settings ###

Legacy API Enabled: –
Force SSL: –
Currency: USD ($)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2
Taxonomies: Product Types: booking (booking)
converge-subscription (converge-subscription)
converge-variable-subscription (converge-variable-subscription)
external (external)
grouped (grouped)
simple (simple)
subscription (subscription)
variable (variable)
variable subscription (variable-subscription)

Taxonomies: Product Visibility: exclude_from_catalog (exclude_from_catalog)
exclude_from_search (exclude_from_search)
exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
instock (instock)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

Connected to WooCommerce.com: –
Enforce Approved Product Download Directories: ✔
HPOS feature enabled: ✔
Order datastore: Automattic\WooCommerce\Internal\DataStores\Orders\OrdersTableDataStore
HPOS data sync enabled: ✔
Enabled Features: analytics
rate_limit_checkout
marketplace
order_attribution
site_visibility_badge
hpos_fts_indexes
remote_logging
email_improvements
point_of_sale
custom_order_tables


### Logging ###

Enabled: ✔
Handler: Automattic\WooCommerce\Internal\Admin\Logging\LogHandlerFileV2
Retention period: 30 days
Level threshold: –
Log directory size: 14 MB

### WC Pages ###

Shop base: #2889 - /shop-2/
Cart: #151 - /cart-block/ -  Contains the woocommerce/cart block
Checkout: #149 - /checkout-2/ -  Contains the woocommerce/checkout block
My account: #2892 - /my-account-2/ -  Contains the [woocommerce_my_account] shortcode
Terms and conditions: ❌ Page not set

### Theme ###

Name: Storefront
Version: 4.6.1
Author URL: https://woocommerce.com/
Child Theme: ❌ – If you are modifying WooCommerce on a parent theme that you did not build personally we recommend using a child theme. See: How to create a child theme
Theme type: Classic theme
WooCommerce Support: ✔

### Templates ###

Overrides: –

### Admin ###

Enabled Features: activity-panels
analytics
product-block-editor
coupons
core-profiler
customize-store
customer-effort-score-tracks
import-products-task
experimental-fashion-sample-products
shipping-smart-defaults
shipping-setting-tour
homescreen
marketing
mobile-app-banner
onboarding
onboarding-tasks
pattern-toolkit-full-composability
product-custom-fields
remote-inbox-notifications
remote-free-extensions
payment-gateway-suggestions
printful
shipping-label-banner
subscriptions
store-alerts
transient-notices
woo-mobile-welcome
wc-pay-promotion
wc-pay-welcome-page
launch-your-store

Disabled Features: product-data-views
experimental-blocks
experimental-iapi-mini-cart
experimental-iapi-runtime
coming-soon-newsletter-template
minified-js
product-pre-publish-modal
settings
async-product-editor-category-field
product-editor-template-system
use-wp-horizon
rest-api-v4

Daily Cron: ✔ Next scheduled: 2025-10-29 18:11:14 +05:30
Options: ✔
Notes: 81
Onboarding: completed

### Braintree (Credit Card) ###

Environment: Sandbox
Tokenization Enabled: ✔
Debug Mode: Save to Log

### Braintree (PayPal) ###

Environment: Sandbox
Tokenization Enabled: ✔
Debug Mode: Save to Log

### Action Scheduler ###

Complete: 196
Oldest: 2025-09-29 12:17:59 +0530
Newest: 2025-10-29 16:15:15 +0530

Failed: 785
Oldest: 2023-02-16 13:17:32 +0530
Newest: 2025-10-28 14:01:59 +0530

Pending: 14
Oldest: 2025-10-29 17:14:55 +0530
Newest: 2026-02-02 11:13:30 +0530


### Status report information ###

Generated at: 2025-10-29 16:55:03 +05:30
```
</details>

</details>

### Isolating the problem

- [x] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [x] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/products/storefront/).
- [x] I can reproduce this bug consistently using the steps above.
